src/bonds.jl

```diff
@@ -9,7 +9,7 @@ export distancebonds,
 """
 	distancebonds( atms ) -> BitMatrix
 
-Returns a matrix of all bonds in `atms`, where Mat[i,j] = 1 if atoms i and j are bonded. 
```

Comment:
assuming these are all from auto-formatting? It complicates the diff a bit, but probably fine

Reply:
Yes, I have my vscode set to trim trailing whitespace when saving files.